ui/src/app/dashboard/views/DiscoveriesList/DiscoveriesList.test.tsx

```diff
@@ -135,4 +135,32 @@ describe("DiscoveriesList", () => {
       .simulate("click");
     expect(wrapper.find("[data-test='delete-discovery']").exists()).toBe(true);
   });
+
+  it("can delete a discovery", () => {
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter
+          initialEntries={[{ pathname: "/dashboard", key: "testKey" }]}
+        >
+          <DiscoveriesList />
+        </MemoryRouter>
+      </Provider>
+    );
+    // Open the action menu.
+    wrapper
+      .find("[data-test='row-menu'] button.p-contextual-menu__toggle")
```

Comment:
It would be safer here to target a `data-test` if we could.

Reply:
There doesn't seem to be a way to pass a `data-test` attribute, but I can pass a class to target.